dao/GlossaryDAO.php

```diff
@@ -0,0 +1,109 @@
+<?php
+namespace Glossary\DAO;
+
+class GlossaryDAO {
+        
+    private $PDOX;
+    private $p;
+
+    public function __construct($PDOX, $p) {
+        $this->PDOX = $PDOX;
+    }
+   
+    // getAllDomains
+    // returns glossary_domains
+    function getDomain($domain_id = 0, $limit = 0, $offset = 0) {   
+        $arr = array();
+        $query = "SELECT *
```

Comment:
for SELECT statements it makes the code more understandable if you add the precise columns you are using in the statement - otherwise it requires a complete understanding of the database table to read the code.

Stating it also makes sure that your code will use the right column names.